apps/studio/components/interfaces/Home/ServiceStatus.tsx

```diff
@@ -105,6 +123,7 @@ const ServiceStatus = () => {
             docsUrl: 'https://supabase.com/docs/guides/functions/troubleshooting',
             isLoading,
             isSuccess: edgeFunctionsStatus?.healthy,
+            logsUrl: '/logs/edge-functions-logs',
```

Comment:
Does this route actually exist?

Reply:
ahh yup! it's in here - just merged it
https://github.com/supabase/supabase/pull/29665